py/desitarget/cuts.py

```diff
@@ -1059,15 +1059,9 @@ def isBGS_colors(rfiberflux=None, gflux=None, rflux=None, zflux=None, w1flux=Non
         primary = np.ones_like(rflux, dtype='?')
     bgs = primary.copy()
     fmc = np.zeros_like(rflux, dtype='?')
-
-    if targtype == 'bright':
-        bgs &= rflux > 10**((22.5-19.5)/2.5)
-    elif targtype == 'faint':
-        bgs &= rflux > 10**((22.5-20.0)/2.5)
-        bgs &= rflux <= 10**((22.5-19.5)/2.5)
-    elif targtype == 'wise':
-        bgs &= rflux > 10**((22.5-20.0)/2.5)
-        bgs &= w1flux*gflux > (zflux*rflux)*10**(-0.2)
+    
+    #the LSLGA galaxies
+    L2 = ((refcat == 'L2') | (refcat == b'L2') | (refcat == 'L2 ') | (refcat == b'L2 '))
```

Comment:
This bit of code will crash the integration tests / simulations. We need to capture the `if refcat is None` case.

Reply:
Hi John. Thanks. I've fixed that.